### PR TITLE
created get methods for doctor profile

### PR DESCRIPTION
**Pull request by serhiidotsyk**

### CODE REVIEW

File: InternetHospital.WebApi/Controllers/DoctorsController.cs [hunk 1]
```diff
@@ -34,6 +40,39 @@ public IActionResult Get([FromRoute] int id)
...
+        }
+
+        [HttpGet("getAvatar")]
+        [Authorize]
+        public async Task<IActionResult> GetAvatar()
+        {
+            var doctorId = User.Identity?.Name;
+            if (doctorId != null)
+            {
+                var doctor = await _userManager.FindByIdAsync(doctorId);// _doctorService.GetAvatar(patientId);
```
Comment (TREN-D) [inline InternetHospital.WebApi/Controllers/DoctorsController.cs]: remove comment

**Review by TREN-D — CHANGES_REQUESTED** [review]
Why all logic in controller istead of service?

File: InternetHospital.WebApi/Controllers/DoctorsController.cs (hunk 1 above)
Comment (serhiidotsyk) [reply InternetHospital.WebApi/Controllers/DoctorsController.cs]: done

File: InternetHospital.WebApi/Controllers/DoctorsController.cs [hunk 2]
```diff
@@ -14,10 +16,12 @@ namespace InternetHospital.WebApi.Controllers
     public class DoctorsController : ControllerBase
     {
         private readonly IDoctorService _doctorService;
+        private readonly IFilesService _uploadingFiles;
 
-        public DoctorsController(IDoctorService service)
+        public DoctorsController(IDoctorService service, UserManager<User> userManager, IFilesService filesService)
```
Comment (TREN-D) [inline InternetHospital.WebApi/Controllers/DoctorsController.cs]: Unused usermanager and filesService

File: InternetHospital.BusinessLogic/Services/DoctorService.cs [hunk 3]
```diff
@@ -140,6 +144,29 @@ public DoctorProfileModel GetDoctorProfile(int userId)
             return doctorModel;
         }
 
+        public async Task<IActionResult> UpdateDoctorAvatar(string doctorId, IFormFile file)
```
Comment (TREN-D) [inline InternetHospital.BusinessLogic/Services/DoctorService.cs]: Dont know if its okay to return IActionResult inside serivce

File: InternetHospital.WebApi/Controllers/DoctorsController.cs (hunk 2 above)
Comment (serhiidotsyk) [reply InternetHospital.WebApi/Controllers/DoctorsController.cs]: done

File: InternetHospital.BusinessLogic/Services/DoctorService.cs (hunk 3 above)
Comment (serhiidotsyk) [reply InternetHospital.BusinessLogic/Services/DoctorService.cs]: changed return type

**Review by TREN-D — APPROVED** [review]

**Review by edward0802 — APPROVED** [review]

**Pull request merged**